matlab/lib/+Ice/UserException.m

```diff
@@ -12,20 +12,6 @@
     methods(Abstract)
         ice_id(obj)
     end
-    methods
-        function obj = UserException(id, msg)
-            obj = obj@Ice.Exception(id, msg);
-        end
-        % ice_getSlicedData - Obtain the SlicedData object that contains the
```

Comment:
Why are we removing this method?

Note it is still mentioned above in `% UserException Methods:`

Reply:
Sorry, forgot to comment on it.

This is part of the left-over preserve-slice simplification that Benoit is working on.

With the simplified 3.8 version (mostly but not fully implemented - in particular, not in MATLAB):
 - user exceptions are always encoded in the Sliced format but are never preserved
 - class encoded in the Slice format are always preserved

That's already implemented in https://github.com/icerpc/icerpc-csharp.

Reply:
Fixed doc-comment.